source/_integrations/shelly.markdown

```diff
@@ -0,0 +1,19 @@
+---
+title: Shelly
+description: Integrate Shelly devices
+ha_category:
+  - Switch
+ha_release: 0.115
+ha_codeowners:
+  - "@balloob"
+ha_domain: shelly
+---
+
+Integrate [Shelly devices](https://shelly.cloud) into Home Assistant. Devices will be automatically discovered when on the same network and can also be manually configured via the integrations page.
+
+Current limitations:
+
+- Only supports firmware 1.8 and later
+- Only support devices without authentication
```

Comment:
This line can be removed after https://github.com/home-assistant/core/pull/39461.
```suggestion
```

Reply:
Removed it upfront, mainly because the parent PR you are referring to has the first approval and is highly likely to end up in the next release.

Make a note to check before the final release to make sure.